x/msgfees/client/cli/cli_test.go

```diff
@@ -70,72 +70,137 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 
 func (s *IntegrationTestSuite) TestMsgFeesTxGovProposals() {
 	testCases := []struct {
-		name         string
-		args         []string
-		expectErr    bool
-		expectErrMsg string
-		expectedCode uint32
+		name          string
+		typeArg       string
+		title         string
+		description   string
+		deposit       string
+		msgType       string
+		additionalFee string
+		recipient     string
+		bips          string
+		expectErr     bool
+		expectErrMsg  string
+		expectedCode  uint32
 	}{
 		{"add msg based fee proposal - valid",
-			[]string{
-				"add",
-				"test add msg based fee",
-				"description",
-				sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String(),
-				"--msg-type=/provenance.metadata.v1.MsgWriteRecordRequest",
-				fmt.Sprintf("--additional-fee=%s", sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(12))).String()),
-			},
+			"add",
+			"test add msg based fee",
+			"description",
+			sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String(),
+			"--msg-type=/provenance.metadata.v1.MsgWriteRecordRequest",
+			fmt.Sprintf("--additional-fee=%s", sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(12))).String()),
+			"",
+			"",
+			false,
+			"",
+			0,
+		},
+		{"add msg based fee proposal with recipient - valid",
```

Comment:
Really really need to use named params when there are this many.

Reply:
I now understand what you mean.  I was adding them from a []string of args to named params in the test struct...that needed to happen as well.  Should be fixed now.